### ISSUE TITLE
change color of axes widget labels

### ISSUE THREAD

**Issue by banesullivan** [akaszynski/vtki#82]
I cannot figure out how to change the color of the labels on the axes widget. Any ideas?

I ask because white text on the `document` plotting theme with a white background does not look good (see below). I'd like to be able to change it to black.

<img width="124" alt="screen shot 2019-02-11 at 2 27 22 pm 1" src="https://user-images.githubusercontent.com/22067021/52594617-2e6d2900-2e09-11e9-8d59-9107d80fb0d9.png">

**Comment by akaszynski** [comment]
This works:
```python
import vtki
sphere = vtki.Sphere()

plotter = vtki.BackgroundPlotter()
plotter.add_mesh(sphere)

axes = plotter.axes_actor
prop_x = axes.GetXAxisCaptionActor2D().GetCaptionTextProperty()
prop_y = axes.GetYAxisCaptionActor2D().GetCaptionTextProperty()
prop_z = axes.GetZAxisCaptionActor2D().GetCaptionTextProperty()

for prop in [prop_x, prop_y, prop_z]:
    prop.SetColor(0, 0, 0)
    prop.SetShadow(False)
```

We might add this as an option for ```add_axes```

**Comment by banesullivan** [comment]
Implemented in 8fb7db7
